opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -0,0 +1,306 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+==== PageSequenceMonitor
+
+The _PageSequenceMonitor_ (PSM) allows OpenNMS to monitor web applications.
+This monitor has several configuration options regarding _IPv4_, _IPv6_ and how to deal with name resolution.
+To add flexibility, the node label and IP address can be passed as variable into the monitor.
+This allows running the monitor with node dependent configuration.
+Beyond testing a web application with a single _URL_ it can also test a path through a web application.
+A test path through an web application can look like this:
+
+. login to a certain web application
+. Execute an action while being logged in
+. Log off
+
+The service is considered as _up_ if all this is working ok.
+If there's an error somewhere, your application will need attention and the service changes the state to _down_.
+
+===== Monitor facts
+
+[options="autowidth"]
+|===
+| Class Name     | `org.opennms.netmgt.poller.monitors.PageSequenceMonitor`
+| Remote Enabled | true
+|===
+
+===== Configuration and Usage
+
+The configuration for this monitor consists of several parts.
+First is the overall configuration for `retries` and `timeouts`.
+These parameters are global for the whole path through the web application.
+
+.Configuration overview of the _PSM_
+image::service-assurance/monitors/03_page-sequence-monitor-config.png[]
+
+The overall layout of the monitor configuration is more complex.
+Additionally, it is possible to configure a page sequence containing a path through a web application.
+
+.Monitor parameters for the PageSequenceMonitor
+[options="header, autowidth"]
+|===
+| Parameter        | Description                                                                           | Required | Default value
+| `timeout`        | Time in milliseconds to wait before assuming that a packet has not responded          | optional | `3000`
+| `retry`          | The number of retries to attempt when a packet fails to respond in the given timeout  | optional | `0`
+| `strict-timeout` | ??                                                                                    | optional | `false`
+| `page-sequence`  | Definition of the page-sequence to execute, see table with _Page Sequence Parameter_  | required | `-`
+| `sequence-retry` | ??                                                                                    | ??       | ??
```

Comment:
This parameter was found in the code but seems to have no function or is unclear what it does? Does it has a default value?